elixir/apps/web/lib/web/live/policies/index.ex

```diff
@@ -88,4 +94,13 @@ defmodule Web.Policies.Index do
     </div>
     """
   end
+
+  def handle_event("delete", %{"id" => id}, socket) do
+    with {:ok, policy} <- Policies.fetch_policy_by_id(id, socket.assigns.subject) do
+      {:ok, _} = Policies.delete_policy(policy, socket.assigns.subject)
+      {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/policies")}
+    else
+      _other -> raise Web.LiveErrors.NotFoundError
+    end
+  end
```

Comment:
This should be an exceptional race condition (somebody deleted while you see it) so crashing here should be fine for now:
```suggestion
  def handle_event("delete", %{"id" => id}, socket) do
      {:ok, _} = Policies.delete_policy(socket.assigns.policy, socket.assigns.subject)
      {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/policies")}
  end
```

Reply:
Sounds good 👍

Reply:
I realized this comment was still open, so I went back and checked on what the behavior was both using the `with` and removing it.  Both seemed to crash, so I went ahead and removed the `with` and `else`.  This should be updated now.